src/pkg/build/sources/conveyorPacker_library_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package sources_test
+
+import (
+	"testing"
+
+	"github.com/sylabs/singularity/src/pkg/build/sources"
+	"github.com/sylabs/singularity/src/pkg/build/types"
+	"github.com/sylabs/singularity/src/pkg/test"
+)
+
+const (
+	libraryURL = "https://library.sylabs.io/"
+	libraryURI = "library://dtrudg/linux/alpine:latest"
+)
+
+// TestLibraryConveyor tests if we can pull an image from singularity hub
+func TestLibraryConveyor(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
+	test.EnsurePrivilege(t)
```

Comment:
The way I'm thinking about it, I think it makes more sense to leave this after the skip check because if we are skipping i don't think we should be doing anything at all in the test case. Thoughts? I agree with the one below and will fix now.

Reply:
agree